pkg/message/message.go

```diff
@@ -29,12 +29,15 @@ type Msg interface {
 	Namespace() string
 }
 
+// Namespace aliases the string type
+type Namespace string
```

Comment:
We convert back and forth all internally... the interface exposes the namespace as a string so I don't think we're gaining anything with this alias.